src/structures/MessageButton.js

```diff
@@ -0,0 +1,166 @@
+'use strict';
+
+const BaseMessageComponent = require('./BaseMessageComponent');
+const { RangeError } = require('../errors');
+const { MessageButtonStyles, MessageComponentTypes } = require('../util/Constants.js');
```

Comment:
```suggestion
const { MessageButtonStyles, MessageComponentTypes } = require('../util/Constants');
```
Just for consistency with the other files.